src/react-components/ui-root.js

```diff
@@ -246,18 +250,22 @@ class UIRoot extends Component {
   };
 
   enterGearVR = async () => {
-    this.exit();
+    if (this.state.availableVREntryTypes.daydream !== VR_DEVICE_AVAILABILITY.yes) {
```

Comment:
Wouldn't it be more clear to test for `isOculusBrowser`?

Reply:
whoa, major bug there. nice catch. it should check for `gearvr` i think

Reply:
the `gearvr` check for `yes` basically implies it is safe to enter VR as is. right now `yes` is only true within Oculus Browser, as per the other change int his app.